src/events/EventsSection.tsx

```diff
@@ -100,6 +179,35 @@ export const EventsSection = () => {
     </>
   );
 
+  const clearSelection = (e: any) => {
```

Comment:
This can be type-safe by specifying the correct type instead of `any`.
```suggestion
  const clearSelection = (e: React.MouseEvent) => {
```

Reply:
that's great! thanks